packages/ios_platform_images/lib/ios_platform_images.dart

```diff
@@ -113,11 +118,11 @@ class IosPlatformImages {
   ///
   /// See [https://developer.apple.com/documentation/uikit/uiimage/1624146-imagenamed?language=objc]
   static ImageProvider load(String name) {
-    Future<Map> loadInfo = _channel.invokeMethod('loadImage', name);
+    Future<Map?> loadInfo = _channel.invokeMapMethod('loadImage', name);
     Completer<Uint8List> bytesCompleter = Completer<Uint8List>();
     Completer<double> scaleCompleter = Completer<double>();
     loadInfo.then((map) {
-      scaleCompleter.complete(map["scale"]);
+      scaleCompleter.complete(map!["scale"]);
       bytesCompleter.complete(map["data"]);
```

Comment:
This will cause an exception if map is `null`. I think it probably needs `completer.completeError` if map is `null`. Which again, it's due to the lack of null safety at the method channel level.